### PR TITLE
Draft: Final countdown

### PR DESCRIPTION
**Pull request by dominikbell**
- Incorporate Erics comments
- reduce resolution of line plots

### CODE REVIEW

**Comment by FrederikSchnack** [comment]
#17 done.

File: literature.bib [hunk 1]
```diff
@@ -102,9 +102,10 @@ @misc{pygyro_code
 }
 
 @misc{pyccel,
-	TITLE={Pyccel - a Python-to-Fortan/C converter, https://github.com/pyccel/pyccel},
-	AUTHOR={Ratnani, Ahmed and others},
-	URL={https://github.com/pyccel/pyccel}
+	TITLE={Pyccel: a Python-to-X transpiler for scientific
+high-performance computing, in preparation},
+	AUTHOR={Bourne, Emily and Güçlü, Yaman and Hadjout, Said and Ratnani, Ahmed},
+	URL={https://joss.theoj.org/papers/8b565b8f36e1d0cccb3c749ff22857e3}
```
Comment (EmilyBourne) [inline literature.bib:108]:
```suggestion
	URL={https://joss.theoj.org/papers/8b565b8f36e1d0cccb3c749ff22857e3},
	year={2023},
	journal={Journal of Open Source Software},
	notes={under review}
```

File: sections/introduction.tex [hunk 2]
```diff
@@ -4,9 +4,9 @@ \section{Introduction}
...
+This paper follows the latter method but replaces the Semi-Lagrangian scheme in the slow subsystem (which does not involve the magnetic field) by an Arakawa scheme. The motivation behind this is to improve conservation properties at the expense of either using a costly implicit time integrator, or an explicit one which is constrained by a CFL condition. For this method we test different orders and boundary conditions. This spatial discretization will be combined with either an implicit Crank-Nicolson integrator or an explicit Runge-Kutta scheme of order 4. The implementation proceeds on top of the \texttt{PyGyro} code \cite{pygyro_code} which is a \textit{Python} implementation of the gyro-kinetic model presented in \cite{Latu_2017} with the aim to replicate test cases of the \texttt{GYSELA} code \cite{Grandgirard_CPC2016}.
```
Comment (EmilyBourne) [inline sections/introduction.tex:9]: I find it a bit strange to say "with the aim to replicate test cases of the \texttt{GYSELA} code \cite{Grandgirard_CPC2016}." when the test cases we are reproducing are no longer present in the GYSELA code.

While this paper is the most up to date reference for GYSELA, it is this paper:
https://www.sciencedirect.com/science/article/pii/S0021999106000155
which presents the test case we are interested in. I am surprised that it is not cited anywhere.

File: literature.bib [hunk 3]
```diff
@@ -102,9 +102,10 @@ @misc{pygyro_code
...
 }
```
Comment (EmilyBourne) [inline literature.bib:109]:
```suggestion
@article{pyccel,
    TITLE={Pyccel: a Python-to-X transpiler for scientific
high-performance computing},
    AUTHOR={Bourne, Emily and Güçlü, Yaman and Hadjout, Said and Ratnani, Ahmed},
    URL={https://joss.theoj.org/papers/8b565b8f36e1d0cccb3c749ff22857e3},
    year={2023},
    journal={Journal of Open Source Software},
    notes={under review}
}
```

File: sections/numerical_experiments.tex [hunk 4]
```diff
@@ -129,23 +128,24 @@ \subsection{Full Gyro-Kinetic Simulations}
 
 The \texttt{PyGyro} code is a \textit{Python 3} library for gyro-kinetic simulations leveraging the acceleration provided by the modules \texttt{Numba} \cite{Numba} or \texttt{Pyccel} \cite{pyccel}. It is highly parallelized using \texttt{MPI} and thus suitable for running even large-scale simulations on computing clusters.
```
Comment (EmilyBourne) [inline sections/numerical_experiments.tex:129]: It is odd to mention `numba` but not `pythran` given that pythran is the preferable choice between the two

File: sections/introduction.tex (hunk 2 above)
Comment (FrederikSchnack) [reply sections/introduction.tex:9]: Maybe it was just a mix-up. I just exchanged the references or should there be an additional change?

Comment (EmilyBourne) [reply sections/introduction.tex:9]: I'm not sure. I imagine that as you talk about GYSELA you wanted/were asked to put the most recent reference to GYSELA.
However the test case that you are talking about is not mentioned in the later paper. Depending on how you want to present this both papers could be relevant. But the older paper is more important for this project

**Comment by FrederikSchnack** [comment]
I added some formatting to the plots and text, removed the SL poloidal plots and rewrote the text accordingly.

**Review by FrederikSchnack — APPROVED** [review]
LGTM. It's in your hands now @dominikbell :gem: :pray:
